pkg/controller/common/reconciler.go

```diff
@@ -87,9 +88,111 @@ func (r *BaseIngressReconciler) ReconcileIngress(ctx context.Context, ci network
 	return nil
 }
 
+func (r *BaseIngressReconciler) reconcileNetworkPolicy(ctx context.Context, ci networkingv1alpha1.IngressAccessor) error {
+	desired := resources.MakeNetworkPolicyAllowAll(ci)
+
+	var networkPolicyList networkingv1.NetworkPolicyList
+	if err := r.Client.List(ctx, &client.ListOptions{}, &networkPolicyList); err != nil {
```

Comment:
Can we limit this list to just the namespace we're interested in?  think the for-loop below is not correct currently because it doesn't check for the correct namespace to find a NetworkPolicy.